### ISSUE TITLE
Change password page does not offer navigation - Dashboard/Account/Sign Out links missing

### ISSUE THREAD

**Issue by EStriegel** [18F/dol-whd-14c#530]
This page:
![image](https://user-images.githubusercontent.com/19496579/34416592-3525d498-ebc2-11e7-8c53-ff8feb59217c.png)

Missing these links:
![image](https://user-images.githubusercontent.com/19496579/37051048-7ad9e6ee-2143-11e8-9c71-4a412f9917e0.png)

**Comment by joebhaktiarDOL** [comment]
@binwang89 please test this page for Section 508 specifically for links that I added.

**Comment by EStriegel** [comment]
@binwang89 Can you confirm this story is complete from an accessibility perspective?  It looks good functionality-wise.

**Comment by binwang89** [comment]
@EStriegel  i have tested the links for this page, it looks good (  Just the links).  however we do have other 508 accessibility issues for this page.  for example #627 (Keyboard Access) and #628 (The skip function does not work properly). I will change this user story to be "Done" and keep other story open till all issues have been fixed.

**Comment by mmurthydol** [comment]
@binwang89 I am moving this story to in-progress. Could you please set up accessibility testing tools on Joe's VM working together with Joe? Let's also discuss the implications of these defects on whether we will fix them or not and may be add this to the running list. After these tasks are done we will close this story.

**Comment by binwang89** [comment]
@mmurthydol @joebhaktiarDOL sent the email with attached sample ESD form 11 for tools  installation.

**Comment by EStriegel** [comment]
@mmurthydol created issue #681. Moving to `Done`
